go/libraries/doltcore/sqle/writer/prolly_table_writer.go

```diff
@@ -39,6 +39,7 @@ type prollyTableWriter struct {
 
 	primary   indexWriter
 	secondary []indexWriter
+	secNames  map[string]int
```

Comment:
let's collapse these into one collection
```suggestion
	secondary map[string]indexWriter
```